### DISCUSSION TITLE
Native support for Two-Factor-Authentication (2FA)

### DISCUSSION THREAD

**Discussion by Ch3ck3rM0n** [jellyfin/jellyfin#11601] [General]
I know this idea is nothing you for some people, but I wanted to give it again to the developer. Would it be possible to implement 2FA into next major v11 release? I know it is possible to sideload a 2FA but a nativ implementation into Jellyfin would be so much nicer and easier to handle for everyone and feels more being in the year 2024.

**Comment by oddstr13** [comment]
[Fider feature request](<https://features.jellyfin.org/posts/26/add-support-for-two-factor-authentication>), please go vote for it there :slightly_smiling_face:

Related feature requests are also listed in the comments there.
As you can see, it has been marked as «PLANNED» there, which means it is on our ever growing long term wishlist.
